docs/topics/wildfly-swarm/docs/reference/fractions/microprofile-jwt.adoc

```diff
@@ -0,0 +1,27 @@
+= MicroProfile JWT RBAC Auth Fraction
+
+
+.Maven Coordinates
+[source,xml]
+----
+<dependency>
+  <groupId>org.wildfly.swarm</groupId>
+  <artifactId>microprofile-jwt</artifactId>
+</dependency>
+----
+
+.Configuration
+
+swarm.microprofile.jwt.default-missing-method-permissions-deny-access:: 
+If a JAX-RS resource has no class-level security metadata, then if this property is set to `true` and at least one resource method has security metadata all other resource methods without security metadata have an implicit `@DenyAll`, otherwise resource methods without security metadata are not secured
+
+swarm.microprofile.jwt.token.exp-grace-period:: 
+The JWT token expiration grace period in seconds 
+
+swarm.microprofile.jwt.token.issued-by:: 
+The URI of the JWT token issuer
+
+swarm.microprofile.jwt.token.signer-pub-key:: 
+The public key of the JWT token signer. Can be prefixed 'file:' or 'classpath:' for key assets, otherwise the key contents are expected
```

Comment:
single quoted values should use backticks

Reply:
Note to self: not easily fixable in our sources.